configs/platforms/amazon-7-aarch64.rb

```diff
@@ -0,0 +1,15 @@
+platform 'amazon-7-aarch64' do |plat|
+  plat.inherit_from_default
+
+  packages = %w[
+    perl-FindBin
+    perl-lib
+    readline-devel
+    systemtap-sdt-devel
+    zlib-devel
+  ]
+
+  plat.provision_with "dnf install -y --allowerasing #{packages.join(' ')}"
```

Comment:
Both the Vanagon default and the 7.x runtime definitions use `yum` instead of `dnf`. I believe `yum` is preferred for this platform.